erpnext/accounts/doctype/payment_reconciliation/test_payment_reconciliation.py

```diff
@@ -1845,6 +1846,78 @@ def test_cr_note_payment_limit_filter(self):
 		self.assertEqual(len(pr.invoices), 1)
 		self.assertEqual(len(pr.payments), 1)
 
+	def test_reconciliation_on_closed_period_payment(self):
+		# create backdated fiscal year
+		first_fy_start_date = frappe.db.get_value("Fiscal Year", {"disabled": 0}, "min(year_start_date)")
+		prev_fy_start_date = add_years(first_fy_start_date, -1)
```

Comment:
While running unit test there is an overlapping of fiscal year, So i've created the fiscal year as early as possible